### PR TITLE
Inhands : the final fuckening

### LINKED ISSUE #4486: power sword lacks an in-hand sprite.
The power sword has no on character in-hand sprite.

### PR DESCRIPTION
**Pull request by alsoandanswer**
- imageadd: "Fixed missing inhands for Kataphract Esword off, Powersword, Null Orb."
  - imageadd: "Made swords contained sprites, and adds some missing sprites. What this means is that you'll be seeing them on your characters a lot more better."
  - imageadd: "New sickle sprite."

Sickle and other sprites made by yours truly.

about 4-5 inhand sprites taken from TG, in food inhands.

It's finally done, hooray! No more merge errors because some guy decided to put in inhands.

dreammaker_pDcGcrCtSG

Look at that. Actually fits in your screen now.
also,

Fixes #4486.

basically, runesword and powersword had the same sprite, so...I just reused it.

DEVNOTE : FOR REVIEWERS
For some icons which are deleted, they probably got renamed/changed.

For files with everything deleted inside, it means I moved them to a different file.
Namely clothing, moved into a clothing file.

Some sword stuff got deleted, they got ALL moved into swords.dmi. Contained sprites!

Removed superfluous waterflower toy.

FINALLY : This doesn't have to be perfect, I'm going through them a second round again file-by-file, in many smaller, bite sized PRs.

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

**Comment by alsoandanswer** [comment]
!bugfix
